src/htmlContent/copy-paste-dialog.html

```diff
@@ -0,0 +1,51 @@
+<div id="myModal" class="modal">
+  <!-- Modal content -->
+  <head>
+    <link rel="stylesheet" type="text/css" href="styles/bramble_overrides.css">
+  </head>
+  <div class="modal-content">
+    <span class="dialog-button" data-button-id="escape">&#10006;</span>
+    <h2 class="model-header">Copying and pasting in Thimble</h2>
+    <p class="model-text">
+      These actions are unavailable via the Edit menu, but you can still use:
+    </p>
+    <table class= "guide">
+    <script>
```

Comment:
You don't want to run a script in this dialog.  Instead, you'll want to follow the pattern here:

https://github.com/adobe/brackets/blob/f028b3ecdf2017498196f39ddbbe4a2364366c92/src/help/HelpCommandHandlers.js#L81

Render variables into your template with Mustache.